### ISSUE TITLE
Modified Elsewhere aria label should be more concise

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#153588]
Testing #153351

Currently the screen reader reads out  "The default value of the setting has also been overridden for the following languages: LANG1, LANG2".

We should always strive towards concisnes for aria labels. So let's go with something like:
"LANG1, LANG2... have overridden this setting"

Similar for other cases. Just try to keep the label as concise as possible :)
